frontend/test/pytest/test_callback.py

```diff
@@ -917,6 +917,63 @@ def expected(x):
 
     # Array(0.4565774, dtype=float64)
 
+def test_scalar_in_array_out_float32_correct():
```

Comment:
For this one, you can use `pytest.parametrize` in the example above and pass in the dtype as a parameter to the test :)